util/util.go

```diff
@@ -166,29 +165,24 @@ func DecodeKey(k string) (*sym.Key, error) {
 
 // SetupDefaultLoggingConfig sets up a standard logging configuration.
 func SetupDefaultLoggingConfig(repoPath string) {
-	lj := &lumberjack.Logger{
-		Filename:   filepath.Join(repoPath, "log"),
-		MaxSize:    10, // megabytes
-		MaxBackups: 3,
-		MaxAge:     30, // days
-	}
-	backendFile := logger.NewLogBackend(lj, "", 0)
-	logger.SetBackend(backendFile)
-	logger.SetFormatter(logger.MustStringFormatter(logging.LogFormats["color"]))
-	logging.SetAllLoggers(logger.ERROR)
+	os.Setenv("GOLOG_LOG_FMT", "color")
+	os.Setenv("GOLOG_FILE", filepath.Join(repoPath, "log", "textile.log"))
```

Comment:
This will write a log to the filesystem, but not using lumberjack and with no log rotation.